iron-component-page.html

```diff
@@ -643,8 +643,8 @@
 
         _analyzerChanged: function() {
           var analyzer = this._analyzer;
-          this._setDocElements(analyzer && analyzer.elements ? analyzer.elements : []);
-          this._setDocBehaviors(analyzer && analyzer.behaviors ? analyzer.behaviors : []);
+          this._setDocElements(analyzer ? analyzer.elementsForFolder(this.base) : []);
+          this._setDocBehaviors(analyzer ? analyzer.elementsForFolder(this.base) : []);
```

Comment:
can we store the result of elementsForFolder in a variable and use it for `_setDocElements` and `_setDocBehaviors`?